Cargo.toml

```diff
@@ -32,8 +32,8 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 """
 
 [dependencies]
-cosmwasm-schema = "2.0.1"
-cosmwasm-std = { version = "2.0.1", features = [
+cosmwasm-schema = "2.1.0"
+cosmwasm-std = { version = "2.1.0", features = [
   "cosmwasm_1_3",
   # Enable this if you only deploy to chains that have CosmWasm 1.4 or higher
   # "cosmwasm_1_4",
```

Comment:
As 1.3 EOLed now we can push this section by one version

Reply:
Nice!